luzer/luzer.c

```diff
@@ -425,6 +462,7 @@ luaL_fuzz(lua_State *L)
 	}
 	lua_pop(L, -1);
 
+	jit_status = luajit_has_enabled_jit(L);
```

Comment:
After the call to the `luaJIT_setmode()` above, this variable will be set to 0.

I suppose it is better to do the following:
1) Check if JIT is enabled.
2) No -> go to 4). Yes -> turn JIT off and do single run without JIT (to collect coverage for example).
3) Turn it back on.
4) The second usual call.